src/payment/data/utils.js

```diff
@@ -288,3 +284,14 @@ export function createCustomRoutine(name, addtlStages, keepDefaults = true) {
     ...addtlStages.map(x => x.toUpperCase()),
   ])(name);
 }
+
+/**
+ * Determines if the MFE should follow the Commerce Coordinator Flows
+ *
+ * This checks a waffle flag and assumes the CommerceCoordinator has set the flag if the order type is right.
+ *
+ * @returns {boolean}
+ */
+export function isCommerceCoordinatorEnabled() {
+  return isWaffleFlagEnabled(WAFFLE_FLAGS.COMMERCE_COORDINATOR_ENABLED);
+}
```

Comment:
Can we call `isWaffleFlagEnabled(WAFFLE_FLAGS.COMMERCE_COORDINATOR_ENABLED)` directly? Worried this is too much wrapping.